src/views/catalog/customize/components/CustomizeSource/ExpandableCustomizeSourceSection.scss

```diff
@@ -0,0 +1,7 @@
+.pf-l-flex {
```

Comment:
Hi :) 
This file is a bit problematic when accessing the PF classes and changing them directly, this will affect the entire plugin and could cause unexpected issues.
Can you try and create new classes instead of changing existing PF classes to achieve the same styling?

Reply:
I can wrap this by some unique custom class name, but the rest needs to remain the same, otherwise we won't achieve the expected result 🤷🏼‍♀️ We do need to override the PF class for this component to get the look and the behavior we need. Another option would be not to use the PF component.

Reply:
wrapping these changes inside a custom classname sound like a great solution :) 
this way we still change what we need but don't affect any other places :100: :1st_place_medal:

Reply:
Well, I have an experience that it may not always work like that, but let's try it out.